actionpack/lib/action_dispatch/request/session.rb

```diff
@@ -233,15 +233,9 @@ def load_for_write!
         def load!
           id, session = @by.load_session @req
           options[:id] = id
-          @delegate.replace(stringify_keys(session))
+          @delegate.replace(session.stringify_keys)
           @loaded = true
         end
-
-        def stringify_keys(other)
-          other.each_with_object({}) { |(key, value), hash|
```

Comment:
This originally autocorrected to the same implementation as `Hash#stringify_keys`, which we can just call directly:

https://github.com/rails/rails/blob/89d1d39c8f668d2fa0aeecb97431095f3f4e7ebc/activesupport/lib/active_support/core_ext/hash/keys.rb#L10-L12